index.html

```diff
@@ -33,6 +35,187 @@ <h2 class="font2">I'm a Software Developer</h2>
+                    <div class="card-works">
+                        <h5 class="title">Profesional Art Printing Data</h5>
+                        <p class="font-body-reg">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <ul class="skill-lists w-247 bg-list">
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button w-100">See Project</button>
+                </div>
+                <div class="card mt-24">
+                    <div class="card-works">
+                        <h5 class="title">Profesional Art Printing Data</h5>
+                        <p class="font-body-reg">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <ul class="skill-lists w-247 bg-list">
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button w-100">See Project</button>
+                </div>
+                <div class="card mt-24">
+                    <div class="card-works">
+                        <h5 class="title">Profesional Art Printing Data</h5>
+                        <p class="font-body-reg">A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                        <ul class="skill-lists w-247 bg-list">
+                            <li>HTML</li>
+                            <li>Bootstrap</li>
+                            <li>Ruby</li>
+                        </ul>
+                    </div>
+                    <button class="button w-100">See Project</button>
+                </div>
+            </div>
+        </section>       
+        <section class="about-me-section-con">
+            <div class="about-me" >
+                <h3 class="title">About me</h3>
+                <p class="font-body-reg m24-0">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+                <button class="button">Get my resume</button>
+            </div>
+            <div class="skills-con">
+                <img class="mb-8" src="assets/mobile/divider.svg" alt="line image">
+                <div class="skill-wrapper d-flex">
+                    <div class="skills">
+                        <div class="skill-title d-flex">
+                             <img src="assets/mobile/diamond.svg" alt="orange diamond">
+                             <h4 class="skill-name">Language</h4>
```

Comment:
- [ ] Kindly consider adding the technology categories ("Languages", "Frameworks" and "Skills")  in a ul, each of them in a li because they are also a list of data